core/domain/learner_progress_services.py

```diff
@@ -1375,7 +1375,8 @@ def get_displayable_topic_summary_dicts(user_id, topic_summaries):
             summary domain objects.
 
     Returns:
-        list(dict). The summary dict objects corresponding to the given summary.
+        list(dict). The summary dict objects corresponding to the given
```

Comment:
Drop "objects", items are either dicts or domain objects but not both.

Reply:
Done.